### PR TITLE
feat: Remove PostgreSQL component

### PR DESCRIPTION
**Pull request by tolusha**
## What does this pull request change?
feat: Remove PostgreSQL component

## What issues does this pull request fix or reference?
https://github.com/eclipse/che/issues/21374

## Specify the version of the product this pull request applies to
7.62.0

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by ibuziuk — COMMENTED** [review]
@tolusha I remember we also had a doc related to external db setup. Not sure if it still exists but would be great to also clean it up

**Review by ibuziuk — COMMENTED** [review]
Not sure if this page was cleaned (including images) - https://www.eclipse.org/che/docs/stable/administration-guide/postgresql/

**Comment by tolusha** [comment]
@ibuziuk 
I've just removed by patching https://github.com/eclipse-che/che-docs/pull/2561/files#diff-85f5122b45d94b158231e52496a9d105a9c751328415936189408f9b966ac298

I can't find any docs with regard to external db setup.

**Review by ibuziuk — APPROVED** [review]

File: modules/administration-guide/pages/installing-che-using-storage-classes.adoc [hunk 1]
```diff
@@ -18,10 +18,9 @@ To configure {prod-short} to use a configured infrastructure storage, install {p
 * A storage type
 * And many others
 
-{prod-short} has two components that require persistent volumes to store data:
+{prod-short} has the only component that require persistent volumes to store data:
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-using-storage-classes.adoc]:
```suggestion
{prod-short} has one component that requires persistent volumes to store data:
```

File: modules/administration-guide/pages/installing-che-using-storage-classes.adoc [hunk 2]
```diff
@@ -18,10 +18,9 @@ To configure {prod-short} to use a configured infrastructure storage, install {p
...
 
- * A PostgreSQL database.
- * A {prod-short} workspaces. {prod-short} workspaces store source code using volumes, for example `/projects` volume.
+  * A {prod-short} workspaces. {prod-short} workspaces store source code using volumes, for example `/projects` volume.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-using-storage-classes.adoc]:
```suggestion
  * A {prod-short} workspace. {prod-short} workspaces store source code using volumes, for example `/projects` volume.
```

**Review by deerskindoll — COMMENTED** [review]
two mini suggestions

**Comment by ibuziuk** [comment]
@deerskindoll thank you for the review, suggestions have been applied :+1:

**Review by deerskindoll — APPROVED** [review]
lgtm

**Pull request merged**